README.md

```diff
@@ -44,6 +44,27 @@ Additionally, JavaScript and CSS has to be included:
 <script src="path/to/component/dist/smartbanner.min.js"></script>
 ```
 
+## Contributing
```

Comment:
Let's move this block to CONTRIBUTING.md and have a link to it from README? 

Github has some automation for it, would make sense.